templates/article/show.html.twig

```diff
@@ -5,14 +5,54 @@
 {% block body %}
     {% if article.featuredImage %}
         <a href="{{ path('article_show', {'slug': article.slug}) }}">
-            <img src="/media/{{ article.featuredImage.filename }}" alt="{{ article.featuredImage.altText }}" loading="lazy" width="350" height="205" >
+            <img src="/media/{{ article.featuredImage.filename }}" alt="{{ article.featuredImage.altText }}"
+                 loading="lazy" width="350" height="205">
         </a>
     {% endif %}
-    <h1 class="article-title">{{ article.title }}</h1>
-    <div id="article-content">
+    <h1 class="article-title font-semibold text-lg">{{ article.title }}</h1>
+    <div id="article-content py-2">
         {{ article.content|raw }}
     </div>
+    <div class="comment py-2">
+        {% for comment in comments %}
+            <div class="comment-card bg-sky-50 rounded-2xl px-4 pt-2 pb-2.5 my-2">
+                <div class="flex justify-between items-center">
+                    <div>
+                        <div class="flex text-base text-gray-800 leading-relaxed">
+                            <div class="font-semibold">
+                                {{ comment.appUser }}
+                            </div>
+                            <div class="text-sm items-center p-1 px-3">
+                                {{ comment.createdAt|date('d-m-Y H:i:s') }}
+                            </div>
+                        </div>
+                        <div class="row-span-2 col-span-2 text-normal leading-snug text-gray-700 md:leading-normal">
+                            {{ comment.content }}
+                        </div>
+                    </div>
+                    {% if is_granted('ROLE_AUTHOR') %}
+                        <a class="text-red-500" href="{{ path('delete_comment', {'id': comment.id}) }}"
+                           onclick="return confirm('Are you sure you want to delete this comment?');">
+                            <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5"
+                                 stroke="currentColor" class="w-6 h-6">
+                                <path stroke-linecap="round" stroke-linejoin="round"
+                                      d="M14.74 9l-.346 9m-4.788 0L9.26 9m9.968-3.21c.342.052.682.107 1.022.166m-1.022-.165L18.16 19.673a2.25 2.25 0 01-2.244 2.077H8.084a2.25 2.25 0 01-2.244-2.077L4.772 5.79m14.456 0a48.108 48.108 0 00-3.478-.397m-12 .562c.34-.059.68-.114 1.022-.165m0 0a48.11 48.11 0 013.478-.397m7.5 0v-.916c0-1.18-.91-2.164-2.09-2.201a51.964 51.964 0 00-3.32 0c-1.18.037-2.09 1.022-2.09 2.201v.916m7.5 0a48.667 48.667 0 00-7.5 0"/>
+                            </svg>
+                        </a>
+                    {% endif %}
+                </div>
 
+            </div>
+        {% endfor %}
+        {% if is_granted('ROLE_AUTHOR') %}
```

Comment:
Why only Author can comment, maybe a `IS_FULLY_AUTHENTICATED` or `IS_AUTHENTICATED_REMEMBERED` will be better

Reply:
@jnadaud, sorry just now saw your comments. I came to home town last week to see my parents, just now back to work place. Can you please let me know, If I need to resolve and push the changes

Reply:
No, it's just a comment. It was just a  little test.

Reply:
Ok, Thanks for the confirmation @FolkenFR